src/fetch_response.js

```diff
@@ -60,6 +60,7 @@ export class FetchResponse {
       } else {
         console.warn('You must set `window.Turbo = Turbo` to automatically process Turbo Stream events with request.js')
       }
+      return
```

Comment:
@excid3 What do you think about moving the else back here as it was before the refactoring? And sorry, I didn't noticed that when reviewed the PR

Reply:
Yeah, that's fine with me. Probably a bit more clear.